cs_integration_tests/integration_test.go

```diff
@@ -0,0 +1,59 @@
+package integration
+
+import (
+	"context"
+	"runtime"
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var cleanupPostgresContainer func()
+
+func TestEverything(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Integration test suite")
+}
+
+var _ = BeforeSuite(func() {
+	runtime.GOMAXPROCS(runtime.NumCPU())
+	var err error
```

Comment:
Move into Go `if` statement

Reply:
I can't because needs to be in scope for use elsewhere in the function